app/create_file.py

```diff
@@ -1 +1,39 @@
-# write your code here
+import os
+import sys
+from datetime import datetime
+
+command = sys.argv
+
+
+def create_file(file_name: str) -> None:
+    with open(file_name, "a") as file:
+        file.write(datetime.now().strftime("%Y-%m-%d, %H:%M:%S") + "\n")
+        line_number = 1
+        while True:
+            line_number += 1
+            line_text = input("Enter content line: ")
+            if line_text == "stop":
+                file.write("\n")
+                break
+            file.write(f"{line_number} {line_text}\n")
+
+
+def creating_file_from_command() -> None:
+    if "-f" in command:
+        if "-d" in command:
+            if command.index("-f") < command.index("-d"):
```

Comment:
As for me must be:
```suggestion
            if command.index("-f") > command.index("-d"):
```
Can you pin screenshots with your testing?